libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -52,8 +52,8 @@ const AP_Param::GroupInfo AC_Fence::var_info[] = {
     // @Param: TYPE
     // @DisplayName: Fence Type
     // @Description: Enabled fence types held as bitmask
-    // @Bitmask{Rover}: 1:Circle,2:Polygon
-    // @Bitmask{Copter, Plane, Sub}: 0:Max altitude,1:Circle,2:Polygon,3:Min altitude
+    // @Bitmask{Rover}: 1:Circle,2:Incusion/Exclusion Circles+Polygons
```

Comment:
Little typo in "inclusion", missing the "l".

Reply:
Fixed.